[bgfx] Build error on x64-linux

Package: bgfx[core,multithreaded,tools]:x64-linux@1.129.8940-496#1

The problem has to do with the fact that glslang::SPVRemapper does not exist anymore and was moved to SPIRV-tools see https://github.com/microsoft/vcpkg/pull/48575 and https://github.com/KhronosGroup/SPIRV-Tools/commit/108b19e5c6979f496deffad4acbe354237afa7d3

The bgfx vcpkg port has a fix_dependencies.patch based on bgfx https://github.com/bkaradzic/bgfx/commit/7a70927d2af7ffcc5cef9c9e8f4c821e31fcb099 that still includes spirv-remap, it was removed in https://github.com/bkaradzic/bgfx/commit/a569829d5081bddb18b0a070801770577b27e06c. The patch does not remove the dependency, it also adds it back to shaderc.cmake, which makes the configure fail because it is not exported by glslang anymore.

I have a PR fixing it, will link to this issue

**Host Environment**

- Host: x64-linux
- Compiler: GNU 15.2.0
-    vcpkg-tool version: 2025-06-20-ef7c0d541124bbdd334a03467e7edb6c3364d199
    vcpkg-scripts version: 0fd59d511d 2026-02-01 (30 seconds ago)

**To Reproduce**

`vcpkg install --x-feature=client`

**Failure logs**

```
-- Using cached bgfx.cmake.v1.129.8940-496.tar.gz
-- Extracting source /home/robin/Documents/projects/wato/vcpkg/downloads/bgfx.cmake.v1.129.8940-496.tar.gz
-- Applying patch fix-dependencies.patch
-- Using source at /home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/src/bgfx-4d65a9dd8a.clean
-- Configuring x64-linux
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:127 (message):
    Command failed: /usr/bin/ninja -v
    Working Directory: /home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      /home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/config-x64-linux-dbg-CMakeCache.txt.log
      /home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/config-x64-linux-rel-CMakeCache.txt.log
      /home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/config-x64-linux-dbg-CMakeConfigureLog.yaml.log
      /home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/config-x64-linux-rel-CMakeConfigureLog.yaml.log
      /home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/config-x64-linux-dbg-ninja.log
      /home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/config-x64-linux-out.log

Call Stack (most recent call first):
  /home/robin/Documents/projects/wato/out/build/unixlike-clang-debug-sccache/vcpkg_installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:269 (vcpkg_execute_required_process)
  /home/robin/Documents/projects/wato/vcpkg_overlay/bgfx/portfile.cmake:45 (vcpkg_cmake_configure)
  scripts/ports.cmake:206 (include)



```

<details><summary>/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/config-x64-linux-out.log</summary>

```
[1/2] "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/bin/cmake" -E chdir "../../x64-linux-dbg" "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/bin/cmake" "/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/src/bgfx-4d65a9dd8a.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=/home/robin/Documents/projects/wato/vcpkg/packages/bgfx_x64-linux/debug" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DBGFX_LIBRARY_TYPE=STATIC" "-DBGFX_AMALGAMATED=ON" "-DBGFX_BUILD_EXAMPLES=OFF" "-DBGFX_OPENGLES_VERSION=30" "-DBGFX_ADDITIONAL_TOOL_PATHS=/home/robin/Documents/projects/wato/out/build/unixlike-clang-debug-sccache/vcpkg_installed/x64-linux/../x64-linux/tools/bgfx" "-DBGFX_BUILD_TOOLS=ON" "-DBGFX_CONFIG_MULTITHREADED=ON" "-DCMAKE_MAKE_PROGRAM=/usr/bin/ninja" "-DCMAKE_SYSTEM_NAME=Linux" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=/home/robin/Documents/projects/wato/vcpkg/scripts/toolchains/linux.cmake" "-DVCPKG_TARGET_TRIPLET=x64-linux" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=external" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=/home/robin/Documents/projects/wato/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=/home/robin/Documents/projects/wato/vcpkg" "-D_VCPKG_INSTALLED_DIR=/home/robin/Documents/projects/wato/out/build/unixlike-clang-debug-sccache/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DBGFX_BUILD_TOOLS=OFF"
-- The C compiler identification is GNU 15.2.0
-- The CXX compiler identification is GNU 15.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Found Stb: /home/robin/Documents/projects/wato/out/build/unixlike-clang-debug-sccache/vcpkg_installed/x64-linux/include
-- Performing Test flag_supported
-- Performing Test flag_supported - Success
-- Found X11: /usr/include
-- Looking for XOpenDisplay in /usr/lib/x86_64-linux-gnu/libX11.so;/usr/lib/x86_64-linux-gnu/libXext.so
-- Looking for XOpenDisplay in /usr/lib/x86_64-linux-gnu/libX11.so;/usr/lib/x86_64-linux-gnu/libXext.so - found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
...
Skipped 40 lines
...
FAILED: [code=1] ../CMakeCache.txt 
"/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/bin/cmake" -E chdir ".." "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/bin/cmake" "/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/src/bgfx-4d65a9dd8a.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=/home/robin/Documents/projects/wato/vcpkg/packages/bgfx_x64-linux" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DBGFX_LIBRARY_TYPE=STATIC" "-DBGFX_AMALGAMATED=ON" "-DBGFX_BUILD_EXAMPLES=OFF" "-DBGFX_OPENGLES_VERSION=30" "-DBGFX_ADDITIONAL_TOOL_PATHS=/home/robin/Documents/projects/wato/out/build/unixlike-clang-debug-sccache/vcpkg_installed/x64-linux/../x64-linux/tools/bgfx" "-DBGFX_BUILD_TOOLS=ON" "-DBGFX_CONFIG_MULTITHREADED=ON" "-DCMAKE_MAKE_PROGRAM=/usr/bin/ninja" "-DCMAKE_SYSTEM_NAME=Linux" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=/home/robin/Documents/projects/wato/vcpkg/scripts/toolchains/linux.cmake" "-DVCPKG_TARGET_TRIPLET=x64-linux" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=external" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=/home/robin/Documents/projects/wato/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=/home/robin/Documents/projects/wato/vcpkg" "-D_VCPKG_INSTALLED_DIR=/home/robin/Documents/projects/wato/out/build/unixlike-clang-debug-sccache/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
-- The C compiler identification is GNU 15.2.0
-- The CXX compiler identification is GNU 15.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Found Stb: /home/robin/Documents/projects/wato/out/build/unixlike-clang-debug-sccache/vcpkg_installed/x64-linux/include
-- Performing Test flag_supported
-- Performing Test flag_supported - Success
-- Found X11: /usr/include
-- Looking for XOpenDisplay in /usr/lib/x86_64-linux-gnu/libX11.so;/usr/lib/x86_64-linux-gnu/libXext.so
-- Looking for XOpenDisplay in /usr/lib/x86_64-linux-gnu/libX11.so;/usr/lib/x86_64-linux-gnu/libXext.so - found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - found
-- Looking for IceConnectionNumber in ICE
-- Looking for IceConnectionNumber in ICE - found
CMake Warning (dev) at /home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/FindOpenGL.cmake:415 (message):
  Policy CMP0072 is not set: FindOpenGL prefers GLVND by default when
  available.  Run "cmake --help-policy CMP0072" for policy details.  Use the
  cmake_policy command to set the policy and suppress this warning.

  FindOpenGL found both a legacy GL library:

    OPENGL_gl_LIBRARY: /usr/lib/x86_64-linux-gnu/libGL.so

  and GLVND libraries for OpenGL and GLX:

    OPENGL_opengl_LIBRARY: /usr/lib/x86_64-linux-gnu/libOpenGL.so
    OPENGL_glx_LIBRARY: /usr/lib/x86_64-linux-gnu/libGLX.so

  OpenGL_GL_PREFERENCE has not been set to "GLVND" or "LEGACY", so for
  compatibility with CMake 3.10 and below the legacy GL library will be used.
Call Stack (most recent call first):
  /home/robin/Documents/projects/wato/vcpkg/scripts/buildsystems/vcpkg.cmake:908 (_find_package)
  cmake/bgfx/bgfx.cmake:185 (find_package)
  cmake/bgfx/CMakeLists.txt:11 (include)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found OpenGL: /usr/lib/x86_64-linux-gnu/libOpenGL.so
-- Configuring done (2.1s)
CMake Error at cmake/bgfx/shaderc.cmake:22 (target_link_libraries):
  Target "shaderc" links to:

    glslang::SPVRemapper

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.

Call Stack (most recent call first):
  cmake/bgfx/CMakeLists.txt:31 (include)


-- Generating done (0.1s)
CMake Warning:
  Manually-specified variables were not used by the project:

    FETCHCONTENT_FULLY_DISCONNECTED
    VCPKG_PLATFORM_TOOLSET
    VCPKG_SET_CHARSET_FLAG
    _VCPKG_ROOT_DIR


CMake Generate step failed.  Build files cannot be regenerated correctly.
ninja: build stopped: subcommand failed.
```
</details>

<details><summary>/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/config-x64-linux-dbg-CMakeCache.txt.log</summary>

```
# This is the CMakeCache file.
# For build in directory: /home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-dbg
# It was generated by CMake: /home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/bin/cmake
# You can edit this file to change values found and used by cmake.
# If you do not want to change any of the values, simply exit the editor.
# If you do want to change a value, simply edit, save, and exit the editor.
# The syntax for the file is as follows:
# KEY:TYPE=VALUE
# KEY is the name of a variable in the cache.
# TYPE is a hint to GUIs for the type of VALUE, DO NOT EDIT TYPE!.
# VALUE is the current value for the KEY.

########################
# EXTERNAL cache entries
########################

//No help, variable specified on the command line.
BGFX_ADDITIONAL_TOOL_PATHS:UNINITIALIZED=/home/robin/Documents/projects/wato/out/build/unixlike-clang-debug-sccache/vcpkg_installed/x64-linux/../x64-linux/tools/bgfx

//Amalgamated bgfx build for faster compilation
BGFX_AMALGAMATED:BOOL=ON

//Build bgfx examples.
BGFX_BUILD_EXAMPLES:BOOL=OFF

//Build bgfx example-common project.
BGFX_BUILD_EXAMPLE_COMMON:BOOL=OFF

//Build bgfx tests.
BGFX_BUILD_TESTS:BOOL=OFF

//Build bgfx tools.
BGFX_BUILD_TOOLS:BOOL=OFF

//Path to custom cmake script to include
BGFX_CMAKE_USER_SCRIPT:STRING=

//Enable gfx debug annotations (default: on in debug)
BGFX_CONFIG_DEBUG_ANNOTATION:BOOL=OFF

//Specify default maximum encoder count (multithreaded only)
BGFX_CONFIG_DEFAULT_MAX_ENCODERS:STRING=

//Specify maximum draw calls
BGFX_CONFIG_MAX_DRAW_CALLS:STRING=

//Specify maximum dynamic index buffer count
BGFX_CONFIG_MAX_DYNAMIC_INDEX_BUFFERS:STRING=

//Specify maximum dynamic vertex buffer count
BGFX_CONFIG_MAX_DYNAMIC_VERTEX_BUFFERS:STRING=

//Specify maximum frame buffers
BGFX_CONFIG_MAX_FRAME_BUFFERS:STRING=

//Specify maximum index buffer count
BGFX_CONFIG_MAX_INDEX_BUFFERS:STRING=

//Specify shader count
BGFX_CONFIG_MAX_SHADERS:STRING=

//Specify maximum texture count
BGFX_CONFIG_MAX_TEXTURES:STRING=

//Specify maximum texture samplers
BGFX_CONFIG_MAX_TEXTURE_SAMPLERS:STRING=

//Specify maximum vertex buffer count
BGFX_CONFIG_MAX_VERTEX_BUFFERS:STRING=

//Specify maximum vertex layouts
BGFX_CONFIG_MAX_VERTEX_LAYOUTS:STRING=

//Specify maximum view count (should be a power of two)
BGFX_CONFIG_MAX_VIEWS:STRING=

//Build bgfx with multithreaded configuration
BGFX_CONFIG_MULTITHREADED:BOOL=ON

//Enable the webgpu renderer
BGFX_CONFIG_RENDERER_WEBGPU:BOOL=OFF

//Specify maximum program count as 2^(program bits)
BGFX_CONFIG_SORT_KEY_NUM_BITS_PROGRAM:STRING=

//Include convenience custom targets.
BGFX_CUSTOM_TARGETS:BOOL=ON

//Location of bgfx.
BGFX_DIR:STRING=/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/src/bgfx-4d65a9dd8a.clean/bgfx

//Create installation target.
BGFX_INSTALL:BOOL=ON

//Linking type for library
BGFX_LIBRARY_TYPE:STRING=STATIC

//Specify minimum OpenGL ES version
BGFX_OPENGLES_VERSION:STRING=30

//Specify minimum opengl version
BGFX_OPENGL_VERSION:STRING=

//Prefix name to add to name of tools (to avoid clashes)
BGFX_TOOLS_PREFIX:STRING=

//Use Wayland backend.
BGFX_WITH_WAYLAND:BOOL=ON

//Location of bimg.
BIMG_DIR:STRING=/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/src/bgfx-4d65a9dd8a.clean/bimg

//No help, variable specified on the command line.
BUILD_SHARED_LIBS:UNINITIALIZED=OFF

//Amalgamated bx build for faster compilation
BX_AMALGAMATED:BOOL=OFF

//Log debug messages (default: on in debug)
BX_CONFIG_DEBUG:BOOL=OFF

//Location of bx.
BX_DIR:STRING=/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/src/bgfx-4d65a9dd8a.clean/bx

//Path to a program.
CMAKE_ADDR2LINE:FILEPATH=/usr/bin/addr2line

//Path to a program.
CMAKE_AR:FILEPATH=/usr/bin/ar

...
Skipped 1308 lines
...
X11_xcb_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_composite_INCLUDE_PATH
X11_xcb_composite_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_composite_LIB
X11_xcb_composite_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_cursor_INCLUDE_PATH
X11_xcb_cursor_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_cursor_LIB
X11_xcb_cursor_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_damage_INCLUDE_PATH
X11_xcb_damage_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_damage_LIB
X11_xcb_damage_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_dpms_INCLUDE_PATH
X11_xcb_dpms_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_dpms_LIB
X11_xcb_dpms_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_dri2_INCLUDE_PATH
X11_xcb_dri2_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_dri2_LIB
X11_xcb_dri2_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_dri3_INCLUDE_PATH
X11_xcb_dri3_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_dri3_LIB
X11_xcb_dri3_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_errors_INCLUDE_PATH
X11_xcb_errors_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_errors_LIB
X11_xcb_errors_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_ewmh_INCLUDE_PATH
X11_xcb_ewmh_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_ewmh_LIB
X11_xcb_ewmh_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_glx_INCLUDE_PATH
X11_xcb_glx_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_glx_LIB
X11_xcb_glx_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_icccm_INCLUDE_PATH
X11_xcb_icccm_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_icccm_LIB
X11_xcb_icccm_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_image_INCLUDE_PATH
X11_xcb_image_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_image_LIB
X11_xcb_image_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_keysyms_INCLUDE_PATH
X11_xcb_keysyms_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_keysyms_LIB
X11_xcb_keysyms_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_present_INCLUDE_PATH
X11_xcb_present_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_present_LIB
X11_xcb_present_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_randr_INCLUDE_PATH
X11_xcb_randr_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_randr_LIB
X11_xcb_randr_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_record_INCLUDE_PATH
X11_xcb_record_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_record_LIB
X11_xcb_record_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_render_INCLUDE_PATH
X11_xcb_render_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_render_LIB
X11_xcb_render_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_render_util_INCLUDE_PATH
X11_xcb_render_util_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_render_util_LIB
X11_xcb_render_util_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_res_INCLUDE_PATH
X11_xcb_res_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_res_LIB
X11_xcb_res_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_screensaver_INCLUDE_PATH
X11_xcb_screensaver_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_screensaver_LIB
X11_xcb_screensaver_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_shape_INCLUDE_PATH
X11_xcb_shape_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_shape_LIB
X11_xcb_shape_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_shm_INCLUDE_PATH
X11_xcb_shm_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_shm_LIB
X11_xcb_shm_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_sync_INCLUDE_PATH
X11_xcb_sync_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_sync_LIB
X11_xcb_sync_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_util_INCLUDE_PATH
X11_xcb_util_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_util_LIB
X11_xcb_util_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xf86dri_INCLUDE_PATH
X11_xcb_xf86dri_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xf86dri_LIB
X11_xcb_xf86dri_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xfixes_INCLUDE_PATH
X11_xcb_xfixes_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xfixes_LIB
X11_xcb_xfixes_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xinerama_INCLUDE_PATH
X11_xcb_xinerama_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xinerama_LIB
X11_xcb_xinerama_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xinput_INCLUDE_PATH
X11_xcb_xinput_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xinput_LIB
X11_xcb_xinput_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xkb_LIB
X11_xcb_xkb_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xrm_INCLUDE_PATH
X11_xcb_xrm_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xrm_LIB
X11_xcb_xrm_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xtest_INCLUDE_PATH
X11_xcb_xtest_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xtest_LIB
X11_xcb_xtest_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xv_INCLUDE_PATH
X11_xcb_xv_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xv_LIB
X11_xcb_xv_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xvmc_INCLUDE_PATH
X11_xcb_xvmc_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xvmc_LIB
X11_xcb_xvmc_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xkbcommon_INCLUDE_PATH
X11_xkbcommon_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xkbcommon_LIB
X11_xkbcommon_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xkbcommon_X11_INCLUDE_PATH
X11_xkbcommon_X11_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xkbcommon_X11_LIB
X11_xkbcommon_X11_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xkbfile_INCLUDE_PATH
X11_xkbfile_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xkbfile_LIB
X11_xkbfile_LIB-ADVANCED:INTERNAL=1
//Making sure VCPKG_MANIFEST_MODE doesn't change
Z_VCPKG_CHECK_MANIFEST_MODE:INTERNAL=OFF
//Vcpkg root directory
Z_VCPKG_ROOT_DIR:INTERNAL=/home/robin/Documents/projects/wato/vcpkg
//CMAKE_INSTALL_PREFIX during last run
_GNUInstallDirs_LAST_CMAKE_INSTALL_PREFIX:INTERNAL=/home/robin/Documents/projects/wato/vcpkg/packages/bgfx_x64-linux/debug
__pkg_config_arguments_PKG_FONTCONFIG:INTERNAL=QUIET;fontconfig
__pkg_config_checked_PKG_FONTCONFIG:INTERNAL=1
//Test flag_supported
flag_supported:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_FONTCONFIG_fontconfig
pkgcfg_lib_PKG_FONTCONFIG_fontconfig-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_FONTCONFIG_freetype
pkgcfg_lib_PKG_FONTCONFIG_freetype-ADVANCED:INTERNAL=1
prefix_result:INTERNAL=/usr/lib/x86_64-linux-gnu
```
</details>

<details><summary>/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/config-x64-linux-rel-CMakeCache.txt.log</summary>

```
# This is the CMakeCache file.
# For build in directory: /home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-rel
# It was generated by CMake: /home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/bin/cmake
# You can edit this file to change values found and used by cmake.
# If you do not want to change any of the values, simply exit the editor.
# If you do want to change a value, simply edit, save, and exit the editor.
# The syntax for the file is as follows:
# KEY:TYPE=VALUE
# KEY is the name of a variable in the cache.
# TYPE is a hint to GUIs for the type of VALUE, DO NOT EDIT TYPE!.
# VALUE is the current value for the KEY.

########################
# EXTERNAL cache entries
########################

//No help, variable specified on the command line.
BGFX_ADDITIONAL_TOOL_PATHS:UNINITIALIZED=/home/robin/Documents/projects/wato/out/build/unixlike-clang-debug-sccache/vcpkg_installed/x64-linux/../x64-linux/tools/bgfx

//Amalgamated bgfx build for faster compilation
BGFX_AMALGAMATED:BOOL=ON

//Build bgfx examples.
BGFX_BUILD_EXAMPLES:BOOL=OFF

//Build bgfx tests.
BGFX_BUILD_TESTS:BOOL=OFF

//Build bgfx tools.
BGFX_BUILD_TOOLS:BOOL=ON

//Build bx binary to c converter.
BGFX_BUILD_TOOLS_BIN2C:BOOL=ON

//Build bgfx geometry tools.
BGFX_BUILD_TOOLS_GEOMETRY:BOOL=ON

//Build bgfx shader tools.
BGFX_BUILD_TOOLS_SHADER:BOOL=ON

//Build bgfx texture tools.
BGFX_BUILD_TOOLS_TEXTURE:BOOL=ON

//Path to custom cmake script to include
BGFX_CMAKE_USER_SCRIPT:STRING=

//Enable gfx debug annotations (default: on in debug)
BGFX_CONFIG_DEBUG_ANNOTATION:BOOL=OFF

//Specify default maximum encoder count (multithreaded only)
BGFX_CONFIG_DEFAULT_MAX_ENCODERS:STRING=

//Specify maximum draw calls
BGFX_CONFIG_MAX_DRAW_CALLS:STRING=

//Specify maximum dynamic index buffer count
BGFX_CONFIG_MAX_DYNAMIC_INDEX_BUFFERS:STRING=

//Specify maximum dynamic vertex buffer count
BGFX_CONFIG_MAX_DYNAMIC_VERTEX_BUFFERS:STRING=

//Specify maximum frame buffers
BGFX_CONFIG_MAX_FRAME_BUFFERS:STRING=

//Specify maximum index buffer count
BGFX_CONFIG_MAX_INDEX_BUFFERS:STRING=

//Specify shader count
BGFX_CONFIG_MAX_SHADERS:STRING=

//Specify maximum texture count
BGFX_CONFIG_MAX_TEXTURES:STRING=

//Specify maximum texture samplers
BGFX_CONFIG_MAX_TEXTURE_SAMPLERS:STRING=

//Specify maximum vertex buffer count
BGFX_CONFIG_MAX_VERTEX_BUFFERS:STRING=

//Specify maximum vertex layouts
BGFX_CONFIG_MAX_VERTEX_LAYOUTS:STRING=

//Specify maximum view count (should be a power of two)
BGFX_CONFIG_MAX_VIEWS:STRING=

//Build bgfx with multithreaded configuration
BGFX_CONFIG_MULTITHREADED:BOOL=ON

//Enable the webgpu renderer
BGFX_CONFIG_RENDERER_WEBGPU:BOOL=OFF

//Specify maximum program count as 2^(program bits)
BGFX_CONFIG_SORT_KEY_NUM_BITS_PROGRAM:STRING=

//Include convenience custom targets.
BGFX_CUSTOM_TARGETS:BOOL=ON

//Location of bgfx.
BGFX_DIR:STRING=/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/src/bgfx-4d65a9dd8a.clean/bgfx

//Create installation target.
BGFX_INSTALL:BOOL=ON

//Linking type for library
BGFX_LIBRARY_TYPE:STRING=STATIC

//Specify minimum OpenGL ES version
BGFX_OPENGLES_VERSION:STRING=30

//Specify minimum opengl version
BGFX_OPENGL_VERSION:STRING=

//Prefix name to add to name of tools (to avoid clashes)
BGFX_TOOLS_PREFIX:STRING=

//Use Wayland backend.
BGFX_WITH_WAYLAND:BOOL=ON

//Location of bimg.
...
Skipped 1376 lines
...
//ADVANCED property for variable: X11_xcb_dri2_INCLUDE_PATH
X11_xcb_dri2_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_dri2_LIB
X11_xcb_dri2_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_dri3_INCLUDE_PATH
X11_xcb_dri3_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_dri3_LIB
X11_xcb_dri3_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_errors_INCLUDE_PATH
X11_xcb_errors_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_errors_LIB
X11_xcb_errors_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_ewmh_INCLUDE_PATH
X11_xcb_ewmh_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_ewmh_LIB
X11_xcb_ewmh_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_glx_INCLUDE_PATH
X11_xcb_glx_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_glx_LIB
X11_xcb_glx_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_icccm_INCLUDE_PATH
X11_xcb_icccm_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_icccm_LIB
X11_xcb_icccm_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_image_INCLUDE_PATH
X11_xcb_image_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_image_LIB
X11_xcb_image_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_keysyms_INCLUDE_PATH
X11_xcb_keysyms_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_keysyms_LIB
X11_xcb_keysyms_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_present_INCLUDE_PATH
X11_xcb_present_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_present_LIB
X11_xcb_present_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_randr_INCLUDE_PATH
X11_xcb_randr_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_randr_LIB
X11_xcb_randr_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_record_INCLUDE_PATH
X11_xcb_record_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_record_LIB
X11_xcb_record_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_render_INCLUDE_PATH
X11_xcb_render_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_render_LIB
X11_xcb_render_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_render_util_INCLUDE_PATH
X11_xcb_render_util_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_render_util_LIB
X11_xcb_render_util_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_res_INCLUDE_PATH
X11_xcb_res_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_res_LIB
X11_xcb_res_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_screensaver_INCLUDE_PATH
X11_xcb_screensaver_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_screensaver_LIB
X11_xcb_screensaver_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_shape_INCLUDE_PATH
X11_xcb_shape_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_shape_LIB
X11_xcb_shape_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_shm_INCLUDE_PATH
X11_xcb_shm_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_shm_LIB
X11_xcb_shm_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_sync_INCLUDE_PATH
X11_xcb_sync_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_sync_LIB
X11_xcb_sync_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_util_INCLUDE_PATH
X11_xcb_util_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_util_LIB
X11_xcb_util_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xf86dri_INCLUDE_PATH
X11_xcb_xf86dri_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xf86dri_LIB
X11_xcb_xf86dri_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xfixes_INCLUDE_PATH
X11_xcb_xfixes_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xfixes_LIB
X11_xcb_xfixes_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xinerama_INCLUDE_PATH
X11_xcb_xinerama_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xinerama_LIB
X11_xcb_xinerama_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xinput_INCLUDE_PATH
X11_xcb_xinput_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xinput_LIB
X11_xcb_xinput_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xkb_LIB
X11_xcb_xkb_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xrm_INCLUDE_PATH
X11_xcb_xrm_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xrm_LIB
X11_xcb_xrm_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xtest_INCLUDE_PATH
X11_xcb_xtest_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xtest_LIB
X11_xcb_xtest_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xv_INCLUDE_PATH
X11_xcb_xv_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xv_LIB
X11_xcb_xv_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xvmc_INCLUDE_PATH
X11_xcb_xvmc_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xcb_xvmc_LIB
X11_xcb_xvmc_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xkbcommon_INCLUDE_PATH
X11_xkbcommon_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xkbcommon_LIB
X11_xkbcommon_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xkbcommon_X11_INCLUDE_PATH
X11_xkbcommon_X11_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xkbcommon_X11_LIB
X11_xkbcommon_X11_LIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xkbfile_INCLUDE_PATH
X11_xkbfile_INCLUDE_PATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: X11_xkbfile_LIB
X11_xkbfile_LIB-ADVANCED:INTERNAL=1
//Making sure VCPKG_MANIFEST_MODE doesn't change
Z_VCPKG_CHECK_MANIFEST_MODE:INTERNAL=OFF
//Vcpkg root directory
Z_VCPKG_ROOT_DIR:INTERNAL=/home/robin/Documents/projects/wato/vcpkg
//CMAKE_INSTALL_PREFIX during last run
_GNUInstallDirs_LAST_CMAKE_INSTALL_PREFIX:INTERNAL=/home/robin/Documents/projects/wato/vcpkg/packages/bgfx_x64-linux
__pkg_config_arguments_PKG_FONTCONFIG:INTERNAL=QUIET;fontconfig
__pkg_config_checked_PKG_FONTCONFIG:INTERNAL=1
//Test flag_supported
flag_supported:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_FONTCONFIG_fontconfig
pkgcfg_lib_PKG_FONTCONFIG_fontconfig-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib_PKG_FONTCONFIG_freetype
pkgcfg_lib_PKG_FONTCONFIG_freetype-ADVANCED:INTERNAL=1
prefix_result:INTERNAL=/usr/lib/x86_64-linux-gnu
```
</details>

<details><summary>/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/config-x64-linux-dbg-CMakeConfigureLog.yaml.log</summary>

```

---
events:
  -
    kind: "message-v1"
    backtrace:
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/CMakeDetermineSystem.cmake:205 (message)"
      - "CMakeLists.txt:12 (project)"
    message: |
      The system is: Linux -  - x86_64
  -
    kind: "message-v1"
    backtrace:
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/CMakeDetermineCompilerId.cmake:17 (message)"
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/CMakeDetermineCompilerId.cmake:64 (__determine_compiler_id_test)"
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/CMakeDetermineCCompiler.cmake:123 (CMAKE_DETERMINE_COMPILER_ID)"
      - "CMakeLists.txt:12 (project)"
    message: |
      Compiling the C compiler identification source file "CMakeCCompilerId.c" succeeded.
      Compiler: /usr/bin/cc 
      Build flags: -fPIC
      Id flags:  
      
      The output was:
      0
      
      
      Compilation of the C compiler identification source "CMakeCCompilerId.c" produced "a.out"
      
      The C compiler identification is GNU, found in:
        /home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-dbg/CMakeFiles/3.31.10/CompilerIdC/a.out
      
  -
    kind: "message-v1"
    backtrace:
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/CMakeDetermineCompilerId.cmake:17 (message)"
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/CMakeDetermineCompilerId.cmake:64 (__determine_compiler_id_test)"
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/CMakeDetermineCXXCompiler.cmake:126 (CMAKE_DETERMINE_COMPILER_ID)"
      - "CMakeLists.txt:12 (project)"
    message: |
      Compiling the CXX compiler identification source file "CMakeCXXCompilerId.cpp" succeeded.
      Compiler: /usr/bin/c++ 
      Build flags: -fPIC
      Id flags:  
      
      The output was:
      0
      
      
      Compilation of the CXX compiler identification source "CMakeCXXCompilerId.cpp" produced "a.out"
      
      The CXX compiler identification is GNU, found in:
        /home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-dbg/CMakeFiles/3.31.10/CompilerIdCXX/a.out
      
  -
    kind: "try_compile-v1"
    backtrace:
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/CMakeDetermineCompilerABI.cmake:74 (try_compile)"
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/CMakeTestCCompiler.cmake:26 (CMAKE_DETERMINE_COMPILER_ABI)"
      - "CMakeLists.txt:12 (project)"
    checks:
      - "Detecting C compiler ABI info"
    directories:
      source: "/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-dbg/CMakeFiles/CMakeScratch/TryCompile-BZJ4Uj"
...
Skipped 767 lines
...
      VCPKG_CRT_LINKAGE: "dynamic"
      VCPKG_CXX_FLAGS: ""
      VCPKG_CXX_FLAGS_DEBUG: ""
      VCPKG_CXX_FLAGS_RELEASE: ""
      VCPKG_C_FLAGS: ""
      VCPKG_C_FLAGS_DEBUG: ""
      VCPKG_C_FLAGS_RELEASE: ""
      VCPKG_INSTALLED_DIR: "/home/robin/Documents/projects/wato/out/build/unixlike-clang-debug-sccache/vcpkg_installed"
      VCPKG_LINKER_FLAGS: ""
      VCPKG_LINKER_FLAGS_DEBUG: ""
      VCPKG_LINKER_FLAGS_RELEASE: ""
      VCPKG_PREFER_SYSTEM_LIBS: "OFF"
      VCPKG_TARGET_ARCHITECTURE: "x64"
      VCPKG_TARGET_TRIPLET: "x64-linux"
      Z_VCPKG_ROOT_DIR: "/home/robin/Documents/projects/wato/vcpkg"
    buildResult:
      variable: "CMAKE_HAVE_REMOVE"
      cached: true
      stdout: |
        Change Dir: '/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-dbg/CMakeFiles/CMakeScratch/TryCompile-lb8ZI4'
        
        Run Build Command(s): /usr/bin/ninja -v cmTC_5f99f
        [1/2] /usr/bin/cc   -fPIC -DCHECK_FUNCTION_EXISTS=remove -fPIE -o CMakeFiles/cmTC_5f99f.dir/CheckFunctionExists.c.o -c /home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-dbg/CMakeFiles/CMakeScratch/TryCompile-lb8ZI4/CheckFunctionExists.c
        [2/2] : && /usr/bin/cc -fPIC -DCHECK_FUNCTION_EXISTS=remove  CMakeFiles/cmTC_5f99f.dir/CheckFunctionExists.c.o -o cmTC_5f99f   && :
        
      exitCode: 0
  -
    kind: "try_compile-v1"
    backtrace:
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/CheckFunctionExists.cmake:93 (try_compile)"
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/FindX11.cmake:740 (check_function_exists)"
      - "/home/robin/Documents/projects/wato/out/build/unixlike-clang-debug-sccache/vcpkg_installed/x64-linux/share/x11/vcpkg-cmake-wrapper.cmake:5 (_find_package)"
      - "/home/robin/Documents/projects/wato/vcpkg/scripts/buildsystems/vcpkg.cmake:862 (include)"
      - "cmake/bgfx/bgfx.cmake:184 (find_package)"
      - "cmake/bgfx/CMakeLists.txt:11 (include)"
    checks:
      - "Looking for shmat"
    directories:
      source: "/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-dbg/CMakeFiles/CMakeScratch/TryCompile-ypcltD"
      binary: "/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-dbg/CMakeFiles/CMakeScratch/TryCompile-ypcltD"
    cmakeVariables:
      CMAKE_C_FLAGS: "-fPIC"
      CMAKE_C_FLAGS_DEBUG: "-g"
      CMAKE_EXE_LINKER_FLAGS: ""
      CMAKE_POSITION_INDEPENDENT_CODE: "ON"
      VCPKG_CHAINLOAD_TOOLCHAIN_FILE: "/home/robin/Documents/projects/wato/vcpkg/scripts/toolchains/linux.cmake"
      VCPKG_CRT_LINKAGE: "dynamic"
      VCPKG_CXX_FLAGS: ""
      VCPKG_CXX_FLAGS_DEBUG: ""
      VCPKG_CXX_FLAGS_RELEASE: ""
      VCPKG_C_FLAGS: ""
      VCPKG_C_FLAGS_DEBUG: ""
      VCPKG_C_FLAGS_RELEASE: ""
      VCPKG_INSTALLED_DIR: "/home/robin/Documents/projects/wato/out/build/unixlike-clang-debug-sccache/vcpkg_installed"
      VCPKG_LINKER_FLAGS: ""
      VCPKG_LINKER_FLAGS_DEBUG: ""
      VCPKG_LINKER_FLAGS_RELEASE: ""
      VCPKG_PREFER_SYSTEM_LIBS: "OFF"
      VCPKG_TARGET_ARCHITECTURE: "x64"
      VCPKG_TARGET_TRIPLET: "x64-linux"
      Z_VCPKG_ROOT_DIR: "/home/robin/Documents/projects/wato/vcpkg"
    buildResult:
      variable: "CMAKE_HAVE_SHMAT"
      cached: true
      stdout: |
        Change Dir: '/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-dbg/CMakeFiles/CMakeScratch/TryCompile-ypcltD'
        
        Run Build Command(s): /usr/bin/ninja -v cmTC_02232
        [1/2] /usr/bin/cc   -fPIC -DCHECK_FUNCTION_EXISTS=shmat -fPIE -o CMakeFiles/cmTC_02232.dir/CheckFunctionExists.c.o -c /home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-dbg/CMakeFiles/CMakeScratch/TryCompile-ypcltD/CheckFunctionExists.c
        [2/2] : && /usr/bin/cc -fPIC -DCHECK_FUNCTION_EXISTS=shmat  CMakeFiles/cmTC_02232.dir/CheckFunctionExists.c.o -o cmTC_02232   && :
        
      exitCode: 0
  -
    kind: "try_compile-v1"
    backtrace:
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/CheckLibraryExists.cmake:78 (try_compile)"
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/FindX11.cmake:750 (check_library_exists)"
      - "/home/robin/Documents/projects/wato/out/build/unixlike-clang-debug-sccache/vcpkg_installed/x64-linux/share/x11/vcpkg-cmake-wrapper.cmake:5 (_find_package)"
      - "/home/robin/Documents/projects/wato/vcpkg/scripts/buildsystems/vcpkg.cmake:862 (include)"
      - "cmake/bgfx/bgfx.cmake:184 (find_package)"
      - "cmake/bgfx/CMakeLists.txt:11 (include)"
    checks:
      - "Looking for IceConnectionNumber in ICE"
    directories:
      source: "/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-dbg/CMakeFiles/CMakeScratch/TryCompile-hqM4J0"
      binary: "/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-dbg/CMakeFiles/CMakeScratch/TryCompile-hqM4J0"
    cmakeVariables:
      CMAKE_C_FLAGS: "-fPIC"
      CMAKE_C_FLAGS_DEBUG: "-g"
      CMAKE_EXE_LINKER_FLAGS: ""
      CMAKE_POSITION_INDEPENDENT_CODE: "ON"
      VCPKG_CHAINLOAD_TOOLCHAIN_FILE: "/home/robin/Documents/projects/wato/vcpkg/scripts/toolchains/linux.cmake"
      VCPKG_CRT_LINKAGE: "dynamic"
      VCPKG_CXX_FLAGS: ""
      VCPKG_CXX_FLAGS_DEBUG: ""
      VCPKG_CXX_FLAGS_RELEASE: ""
      VCPKG_C_FLAGS: ""
      VCPKG_C_FLAGS_DEBUG: ""
      VCPKG_C_FLAGS_RELEASE: ""
      VCPKG_INSTALLED_DIR: "/home/robin/Documents/projects/wato/out/build/unixlike-clang-debug-sccache/vcpkg_installed"
      VCPKG_LINKER_FLAGS: ""
      VCPKG_LINKER_FLAGS_DEBUG: ""
      VCPKG_LINKER_FLAGS_RELEASE: ""
      VCPKG_PREFER_SYSTEM_LIBS: "OFF"
      VCPKG_TARGET_ARCHITECTURE: "x64"
      VCPKG_TARGET_TRIPLET: "x64-linux"
      Z_VCPKG_ROOT_DIR: "/home/robin/Documents/projects/wato/vcpkg"
    buildResult:
      variable: "CMAKE_LIB_ICE_HAS_ICECONNECTIONNUMBER"
      cached: true
      stdout: |
        Change Dir: '/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-dbg/CMakeFiles/CMakeScratch/TryCompile-hqM4J0'
        
        Run Build Command(s): /usr/bin/ninja -v cmTC_af405
        [1/2] /usr/bin/cc   -fPIC -DCHECK_FUNCTION_EXISTS=IceConnectionNumber -fPIE -o CMakeFiles/cmTC_af405.dir/CheckFunctionExists.c.o -c /home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-dbg/CMakeFiles/CMakeScratch/TryCompile-hqM4J0/CheckFunctionExists.c
        [2/2] : && /usr/bin/cc -fPIC -DCHECK_FUNCTION_EXISTS=IceConnectionNumber  CMakeFiles/cmTC_af405.dir/CheckFunctionExists.c.o -o cmTC_af405  -lICE && :
        
      exitCode: 0
...
```
</details>

<details><summary>/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/config-x64-linux-rel-CMakeConfigureLog.yaml.log</summary>

```

---
events:
  -
    kind: "message-v1"
    backtrace:
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/CMakeDetermineSystem.cmake:205 (message)"
      - "CMakeLists.txt:12 (project)"
    message: |
      The system is: Linux -  - x86_64
  -
    kind: "message-v1"
    backtrace:
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/CMakeDetermineCompilerId.cmake:17 (message)"
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/CMakeDetermineCompilerId.cmake:64 (__determine_compiler_id_test)"
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/CMakeDetermineCCompiler.cmake:123 (CMAKE_DETERMINE_COMPILER_ID)"
      - "CMakeLists.txt:12 (project)"
    message: |
      Compiling the C compiler identification source file "CMakeCCompilerId.c" succeeded.
      Compiler: /usr/bin/cc 
      Build flags: -fPIC
      Id flags:  
      
      The output was:
      0
      
      
      Compilation of the C compiler identification source "CMakeCCompilerId.c" produced "a.out"
      
      The C compiler identification is GNU, found in:
        /home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-rel/CMakeFiles/3.31.10/CompilerIdC/a.out
      
  -
    kind: "message-v1"
    backtrace:
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/CMakeDetermineCompilerId.cmake:17 (message)"
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/CMakeDetermineCompilerId.cmake:64 (__determine_compiler_id_test)"
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/CMakeDetermineCXXCompiler.cmake:126 (CMAKE_DETERMINE_COMPILER_ID)"
      - "CMakeLists.txt:12 (project)"
    message: |
      Compiling the CXX compiler identification source file "CMakeCXXCompilerId.cpp" succeeded.
      Compiler: /usr/bin/c++ 
      Build flags: -fPIC
      Id flags:  
      
      The output was:
      0
      
      
      Compilation of the CXX compiler identification source "CMakeCXXCompilerId.cpp" produced "a.out"
      
      The CXX compiler identification is GNU, found in:
        /home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-rel/CMakeFiles/3.31.10/CompilerIdCXX/a.out
      
  -
    kind: "try_compile-v1"
    backtrace:
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/CMakeDetermineCompilerABI.cmake:74 (try_compile)"
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/CMakeTestCCompiler.cmake:26 (CMAKE_DETERMINE_COMPILER_ABI)"
      - "CMakeLists.txt:12 (project)"
    checks:
      - "Detecting C compiler ABI info"
    directories:
      source: "/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-rel/CMakeFiles/CMakeScratch/TryCompile-8vmvWg"
...
Skipped 767 lines
...
      VCPKG_CRT_LINKAGE: "dynamic"
      VCPKG_CXX_FLAGS: ""
      VCPKG_CXX_FLAGS_DEBUG: ""
      VCPKG_CXX_FLAGS_RELEASE: ""
      VCPKG_C_FLAGS: ""
      VCPKG_C_FLAGS_DEBUG: ""
      VCPKG_C_FLAGS_RELEASE: ""
      VCPKG_INSTALLED_DIR: "/home/robin/Documents/projects/wato/out/build/unixlike-clang-debug-sccache/vcpkg_installed"
      VCPKG_LINKER_FLAGS: ""
      VCPKG_LINKER_FLAGS_DEBUG: ""
      VCPKG_LINKER_FLAGS_RELEASE: ""
      VCPKG_PREFER_SYSTEM_LIBS: "OFF"
      VCPKG_TARGET_ARCHITECTURE: "x64"
      VCPKG_TARGET_TRIPLET: "x64-linux"
      Z_VCPKG_ROOT_DIR: "/home/robin/Documents/projects/wato/vcpkg"
    buildResult:
      variable: "CMAKE_HAVE_REMOVE"
      cached: true
      stdout: |
        Change Dir: '/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-rel/CMakeFiles/CMakeScratch/TryCompile-IUPuaF'
        
        Run Build Command(s): /usr/bin/ninja -v cmTC_b84b8
        [1/2] /usr/bin/cc   -fPIC -DCHECK_FUNCTION_EXISTS=remove -fPIE -o CMakeFiles/cmTC_b84b8.dir/CheckFunctionExists.c.o -c /home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-rel/CMakeFiles/CMakeScratch/TryCompile-IUPuaF/CheckFunctionExists.c
        [2/2] : && /usr/bin/cc -fPIC -DCHECK_FUNCTION_EXISTS=remove  CMakeFiles/cmTC_b84b8.dir/CheckFunctionExists.c.o -o cmTC_b84b8   && :
        
      exitCode: 0
  -
    kind: "try_compile-v1"
    backtrace:
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/CheckFunctionExists.cmake:93 (try_compile)"
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/FindX11.cmake:740 (check_function_exists)"
      - "/home/robin/Documents/projects/wato/out/build/unixlike-clang-debug-sccache/vcpkg_installed/x64-linux/share/x11/vcpkg-cmake-wrapper.cmake:5 (_find_package)"
      - "/home/robin/Documents/projects/wato/vcpkg/scripts/buildsystems/vcpkg.cmake:862 (include)"
      - "cmake/bgfx/bgfx.cmake:184 (find_package)"
      - "cmake/bgfx/CMakeLists.txt:11 (include)"
    checks:
      - "Looking for shmat"
    directories:
      source: "/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-rel/CMakeFiles/CMakeScratch/TryCompile-fB28xR"
      binary: "/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-rel/CMakeFiles/CMakeScratch/TryCompile-fB28xR"
    cmakeVariables:
      CMAKE_C_FLAGS: "-fPIC"
      CMAKE_C_FLAGS_DEBUG: "-g"
      CMAKE_EXE_LINKER_FLAGS: ""
      CMAKE_POSITION_INDEPENDENT_CODE: "ON"
      VCPKG_CHAINLOAD_TOOLCHAIN_FILE: "/home/robin/Documents/projects/wato/vcpkg/scripts/toolchains/linux.cmake"
      VCPKG_CRT_LINKAGE: "dynamic"
      VCPKG_CXX_FLAGS: ""
      VCPKG_CXX_FLAGS_DEBUG: ""
      VCPKG_CXX_FLAGS_RELEASE: ""
      VCPKG_C_FLAGS: ""
      VCPKG_C_FLAGS_DEBUG: ""
      VCPKG_C_FLAGS_RELEASE: ""
      VCPKG_INSTALLED_DIR: "/home/robin/Documents/projects/wato/out/build/unixlike-clang-debug-sccache/vcpkg_installed"
      VCPKG_LINKER_FLAGS: ""
      VCPKG_LINKER_FLAGS_DEBUG: ""
      VCPKG_LINKER_FLAGS_RELEASE: ""
      VCPKG_PREFER_SYSTEM_LIBS: "OFF"
      VCPKG_TARGET_ARCHITECTURE: "x64"
      VCPKG_TARGET_TRIPLET: "x64-linux"
      Z_VCPKG_ROOT_DIR: "/home/robin/Documents/projects/wato/vcpkg"
    buildResult:
      variable: "CMAKE_HAVE_SHMAT"
      cached: true
      stdout: |
        Change Dir: '/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-rel/CMakeFiles/CMakeScratch/TryCompile-fB28xR'
        
        Run Build Command(s): /usr/bin/ninja -v cmTC_364a3
        [1/2] /usr/bin/cc   -fPIC -DCHECK_FUNCTION_EXISTS=shmat -fPIE -o CMakeFiles/cmTC_364a3.dir/CheckFunctionExists.c.o -c /home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-rel/CMakeFiles/CMakeScratch/TryCompile-fB28xR/CheckFunctionExists.c
        [2/2] : && /usr/bin/cc -fPIC -DCHECK_FUNCTION_EXISTS=shmat  CMakeFiles/cmTC_364a3.dir/CheckFunctionExists.c.o -o cmTC_364a3   && :
        
      exitCode: 0
  -
    kind: "try_compile-v1"
    backtrace:
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/CheckLibraryExists.cmake:78 (try_compile)"
      - "/home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/share/cmake-3.31/Modules/FindX11.cmake:750 (check_library_exists)"
      - "/home/robin/Documents/projects/wato/out/build/unixlike-clang-debug-sccache/vcpkg_installed/x64-linux/share/x11/vcpkg-cmake-wrapper.cmake:5 (_find_package)"
      - "/home/robin/Documents/projects/wato/vcpkg/scripts/buildsystems/vcpkg.cmake:862 (include)"
      - "cmake/bgfx/bgfx.cmake:184 (find_package)"
      - "cmake/bgfx/CMakeLists.txt:11 (include)"
    checks:
      - "Looking for IceConnectionNumber in ICE"
    directories:
      source: "/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-rel/CMakeFiles/CMakeScratch/TryCompile-frpaYu"
      binary: "/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-rel/CMakeFiles/CMakeScratch/TryCompile-frpaYu"
    cmakeVariables:
      CMAKE_C_FLAGS: "-fPIC"
      CMAKE_C_FLAGS_DEBUG: "-g"
      CMAKE_EXE_LINKER_FLAGS: ""
      CMAKE_POSITION_INDEPENDENT_CODE: "ON"
      VCPKG_CHAINLOAD_TOOLCHAIN_FILE: "/home/robin/Documents/projects/wato/vcpkg/scripts/toolchains/linux.cmake"
      VCPKG_CRT_LINKAGE: "dynamic"
      VCPKG_CXX_FLAGS: ""
      VCPKG_CXX_FLAGS_DEBUG: ""
      VCPKG_CXX_FLAGS_RELEASE: ""
      VCPKG_C_FLAGS: ""
      VCPKG_C_FLAGS_DEBUG: ""
      VCPKG_C_FLAGS_RELEASE: ""
      VCPKG_INSTALLED_DIR: "/home/robin/Documents/projects/wato/out/build/unixlike-clang-debug-sccache/vcpkg_installed"
      VCPKG_LINKER_FLAGS: ""
      VCPKG_LINKER_FLAGS_DEBUG: ""
      VCPKG_LINKER_FLAGS_RELEASE: ""
      VCPKG_PREFER_SYSTEM_LIBS: "OFF"
      VCPKG_TARGET_ARCHITECTURE: "x64"
      VCPKG_TARGET_TRIPLET: "x64-linux"
      Z_VCPKG_ROOT_DIR: "/home/robin/Documents/projects/wato/vcpkg"
    buildResult:
      variable: "CMAKE_LIB_ICE_HAS_ICECONNECTIONNUMBER"
      cached: true
      stdout: |
        Change Dir: '/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-rel/CMakeFiles/CMakeScratch/TryCompile-frpaYu'
        
        Run Build Command(s): /usr/bin/ninja -v cmTC_39206
        [1/2] /usr/bin/cc   -fPIC -DCHECK_FUNCTION_EXISTS=IceConnectionNumber -fPIE -o CMakeFiles/cmTC_39206.dir/CheckFunctionExists.c.o -c /home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-rel/CMakeFiles/CMakeScratch/TryCompile-frpaYu/CheckFunctionExists.c
        [2/2] : && /usr/bin/cc -fPIC -DCHECK_FUNCTION_EXISTS=IceConnectionNumber  CMakeFiles/cmTC_39206.dir/CheckFunctionExists.c.o -o cmTC_39206  -lICE && :
        
      exitCode: 0
...
```
</details>

<details><summary>/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/config-x64-linux-dbg-ninja.log</summary>

```
# CMAKE generated file: DO NOT EDIT!
# Generated by "Ninja" Generator, CMake Version 3.31

# This file contains all the build statements describing the
# compilation DAG.

# =============================================================================
# Write statements declared in CMakeLists.txt:
# 
# Which is the root file.
# =============================================================================

# =============================================================================
# Project: bgfx
# Configurations: Debug
# =============================================================================

#############################################
# Minimal version of Ninja required by this file

ninja_required_version = 1.5


#############################################
# Set configuration variable for custom commands.

CONFIGURATION = Debug
# =============================================================================
# Include auxiliary files.


#############################################
# Include rules file.

include CMakeFiles/rules.ninja

# =============================================================================

#############################################
# Logical path to working directory; prefix for absolute paths.

cmake_ninja_workdir = /home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-dbg/

#############################################
# Utility command for edit_cache

build CMakeFiles/edit_cache.util: CUSTOM_COMMAND
  COMMAND = cd /home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-dbg && /home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/bin/ccmake -S/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/src/bgfx-4d65a9dd8a.clean -B/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-dbg
  DESC = Running CMake cache editor...
  pool = console
  restat = 1

build edit_cache: phony CMakeFiles/edit_cache.util


#############################################
# Utility command for rebuild_cache

build CMakeFiles/rebuild_cache.util: CUSTOM_COMMAND
  COMMAND = cd /home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-dbg && /home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/bin/cmake --regenerate-during-build -S/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/src/bgfx-4d65a9dd8a.clean -B/home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-dbg
  DESC = Running CMake to regenerate build system...
  pool = console
  restat = 1

build rebuild_cache: phony CMakeFiles/rebuild_cache.util


#############################################
# Utility command for list_install_components

build list_install_components: phony


#############################################
# Utility command for install

build CMakeFiles/install.util: CUSTOM_COMMAND all
  COMMAND = cd /home/robin/Documents/projects/wato/vcpkg/buildtrees/bgfx/x64-linux-dbg && /home/robin/Documents/projects/wato/vcpkg/downloads/tools/cmake-3.31.10-linux/cmake-3.31.10-linux-x86_64/bin/cmake -P cmake_install.cmake
  DESC = Install the project...
  pool = console
  restat = 1

build install: phony CMakeFiles/install.util


#############################################
# Utility command for install/local

build CMakeFiles/install/local.util: CUSTOM_COMMAND all
...
Skipped 1259 lines
...


#############################################
# Clean all the built files.

build clean: CLEAN


#############################################
# Print all primary targets available.

build help: HELP


#############################################
# Make the all target the default.

default all
```
</details>

**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "name": "wato",
  "dependencies": [
    "zlib",
    "libsodium",
    "spdlog",
    "reactphysics3d",
    "safeint",
    "glaze",
    "cpr"
  ],
  "features": {
    "client": {
      "description": "WaTo game client",
      "dependencies": [
        {
          "name": "glfw3",
          "features": [
            {
              "name": "wayland",
              "platform": "linux"
            },
            {
              "name": "x11",
              "platform": "linux"
            }
          ]
        },
        {
          "name": "bgfx",
          "default-features": false,
          "features": [
            "multithreaded",
            "tools"
          ],
          "host": true,
          "$reason": "tools are used on host, can't compile on non native"
        },
        {
          "name": "bgfx",
          "features": [
            "multithreaded"
          ]
        },
        "assimp",
        "imgui"
      ]
    },
    "tests": {
      "description": "Build tests",
      "dependencies": [
        "bgfx"
      ]
    }
  }
}

```
</details>

--- a/ports/bgfx/fix-dependencies.patch
+++ b/ports/bgfx/fix-dependencies.patch
@@ -100,7 +100,7 @@ diff --git a/bgfx/tools/shaderc/shaderc_metal.cpp b/bgfx/tools/shaderc/shaderc_m
 index 9f073b9..e8fd208 100644
 --- a/bgfx/tools/shaderc/shaderc_metal.cpp
 +++ b/bgfx/tools/shaderc/shaderc_metal.cpp
-@@ -20,11 +20,11 @@ BX_PRAGMA_DIAGNOSTIC_IGNORED_CLANG_GCC("-Wshadow") // warning: declaration of 'u
+@@ -20,11 +20,10 @@ BX_PRAGMA_DIAGNOSTIC_IGNORED_CLANG_GCC("-Wshadow") // warning: declaration of 'u
  #include <spirv_reflect.hpp>
  
  #define ENABLE_OPT 1
@@ -112,7 +112,6 @@ index 9f073b9..e8fd208 100644
 +#include <glslang/Public/ShaderLang.h>
 +#include <glslang/Include/ResourceLimits.h>
 +#include <glslang/SPIRV/GlslangToSpv.h>
-+#include <glslang/SPIRV/SPVRemapper.h>
 +#include <glslang/SPIRV/SpvTools.h>
  #include <spirv-tools/optimizer.hpp>
  BX_PRAGMA_DIAGNOSTIC_POP()
@@ -121,7 +120,7 @@ diff --git a/bgfx/tools/shaderc/shaderc_spirv.cpp b/bgfx/tools/shaderc/shaderc_s
 index f7910de..a844cc0 100644
 --- a/bgfx/tools/shaderc/shaderc_spirv.cpp
 +++ b/bgfx/tools/shaderc/shaderc_spirv.cpp
-@@ -20,11 +20,11 @@ BX_PRAGMA_DIAGNOSTIC_IGNORED_CLANG_GCC("-Wshadow") // warning: declaration of 'u
+@@ -20,11 +20,10 @@ BX_PRAGMA_DIAGNOSTIC_IGNORED_CLANG_GCC("-Wshadow") // warning: declaration of 'u
  #include <spirv_reflect.hpp>
  
  #define ENABLE_OPT 1
@@ -132,7 +131,6 @@ index f7910de..a844cc0 100644
 -#include <SPIRV/SpvTools.h>
 +#include <glslang/Public/ShaderLang.h>
 +#include <glslang/Include/ResourceLimits.h>
-+#include <glslang/SPIRV/SPVRemapper.h>
 +#include <glslang/SPIRV/GlslangToSpv.h>
 +#include <glslang/SPIRV/SpvTools.h>
  #include <spirv-tools/optimizer.hpp>
@@ -335,7 +333,7 @@ index 0f50eab..8b13b28 100644
 -			bgfx-vertexlayout
 -			fcpp
 -			glslang
-+			glslang::glslang glslang::glslang-default-resource-limits glslang::SPIRV glslang::SPVRemapper
++			glslang::glslang glslang::glslang-default-resource-limits glslang::SPIRV
  			glsl-optimizer
 -			spirv-opt
 -			spirv-cross

--- a/ports/bgfx/vcpkg.json
+++ b/ports/bgfx/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "bgfx",
   "version": "1.129.8940-496",
+  "port-version": 1,
   "maintainers": "Sandy Carter <bwrsandman@users.noreply.github.com>",
   "description": "Cross-platform, graphics API agnostic, Bring Your Own Engine/Framework style rendering library.",
   "homepage": "https://bkaradzic.github.io/bgfx/overview.html",

--- a/versions/b-/bgfx.json
+++ b/versions/b-/bgfx.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "4bf01fdbfc053cd0a55f0794ac38c28ddb37d04a",
+      "version": "1.129.8940-496",
+      "port-version": 1
+    },
+    {
       "git-tree": "0649ad4eb640389c105dbac3a302e9c27e0a04b0",
       "version": "1.129.8940-496",
       "port-version": 0

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -718,7 +718,7 @@
     },
     "bgfx": {
       "baseline": "1.129.8940-496",
-      "port-version": 0
+      "port-version": 1
     },
     "bigint": {
       "baseline": "2010.04.30",

